versions.props

```diff
@@ -19,7 +19,7 @@ org.apache.arrow:arrow-memory-netty = 2.0.0
 com.github.stephenc.findbugs:findbugs-annotations = 1.3.9-1
 software.amazon.awssdk:* = 2.15.7
 org.scala-lang:scala-library = 2.12.10
-org.projectnessie:* = 0.5.1
+org.projectnessie:* = 0.6.1
 javax.ws.rs:javax.ws.rs-api = 2.1.1
 io.quarkus:* = 1.13.1.Final
```

Comment:
does `0.6.1` have a new version of quarkus?

Reply:
Ah! Good catch! Updated it.